reference/src/representation/pointers.md

```diff
@@ -0,0 +1,43 @@
+# Representation of reference and pointer types
+
+### Terminology
+
+Reference types are types of the form `&T` or `&mut T`.
+
+Raw pointer types are types of the form `*const T` or `*mut T`.
+
+### Representation
+
+The alignment of reference and raw pointer types is the word size.
+
+The sizes of `&T`, `&mut T`, `*const T` and `*mut T` are the same,
+and are at least one word.
+
+* If `T` is a trait, then the size of `&T` is two words.
```

Comment:
Should be using `dyn T`. Also `T` is a *type* anywhere else, maybe just use `&dyn Trait`?

Reply:
There are a few other places where `T` is a trait, e.g. `&(dyn T + U)`. I put in all the `dyn`s I could find.